### PR TITLE
Make Cypher Manual AuraDB Business Critical aware

### PR DESCRIPTION
**Pull request by JPryce-Aklundh**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1072.surge.sh

Updated pages:
- [Cypher and Aura](https://neo4j-docs-cypher-1072.surge.sh/cypher-manual/5/introduction/cypher-aura)

File: modules/ROOT/pages/introduction/cypher-aura.adoc [hunk 1]
```diff
@@ -13,41 +13,35 @@ AuraDB is a graph database service for developers building intelligent applicati
 AuraDB is available on the following tiers:
 
 * AuraDB Free
-* AuraDB Pro
+* AuraDB Professional
+* AuraDB Business Critical
 * AuraDB Enterprise
```
Comment (AlexicaWright) [inline modules/ROOT/pages/introduction/cypher-aura.adoc]:
```suggestion
* AuraDB Virtual Dedicated Cloud
```

**Review by AlexicaWright — COMMENTED** [review]
One small change needed, otherwise LGTM!

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Pull request merged**